### ISSUE TITLE
Wrong sleep syntaksis

### ISSUE THREAD

**Issue by 0Karakurt0** [topkecleon/telegram-bot-bash#172]
```
Try 'sleep --help' for more information.
sleep: invalid time interval '2,5'
```
This is what I've got when launched bot in debug mode. There should be dot instead

**Comment by gnadelwartz** [comment]
hm, this looks like a LANG problem, whats your language setting?

**Comment by gnadelwartz** [comment]
I guess it's because locale `C` is not installed on your system and you use a local where `,` is the decimal seperator like in german or other european languages. This should fix it for the next release: fe1fb757481f36233f3ad30bb394555c1ca4c488 

```bash
# to fix it temporary in your bot change:
_round_float() {
        local digit="$2";
...
# to:

_round_float() {
        local digit="0";
...
```
